test-resources/swift/cleanup_rules/input/SampleClass.swift

```diff
@@ -39,4 +39,99 @@ class SampleClass {
         isEnabled = f2() || placeholder_false 
         isEnabled = v2 || placeholder_false 
     }
+
+    func checkAndFalse() {
+        isEnabled = placeholder_false && f1()
+        isEnabled = placeholder_false && v1
+        isEnabled = f2() && placeholder_false 
+        isEnabled = v2 && placeholder_false 
+    }
+
+    func checkIfFalse() {
+        if !TestEnum.stale_flag_one.isEnabled && abc {
+            f1()
+        }
+
+        if !TestEnum.stale_flag_one.isEnabled && abc {
+            f1()
+        } else {
+            f2()
+        }
+
+        if !TestEnum.stale_flag_one.isEnabled && abc {
+            f1()
+        } else if v1 {
+            f2()
+        } else {
+            f3()
+        }
+
+        if v1 {
+            f1()
+        } else if !TestEnum.stale_flag_one.isEnabled && abc {
+            f2()
+        } else {
+            f3()
+        }
+
+        if v1 {
+            f1()
+        } else if !TestEnum.stale_flag_one.isEnabled && abc {
+            f2()
+        } else if v2 {
+            f3()
+        } else {
+            f4()
+        }
+
+        if v1 {
+            f1()
+        } else if !TestEnum.stale_flag_one.isEnabled && abc {
+            f2()
+        }
+    }
+
+    func checkIfLetFalse() {
+        if let v1 = v1, !TestEnum.stale_flag_one.isEnabled && abc {
+            f1()
+        }
+
+        if let v1 = v1, !TestEnum.stale_flag_one.isEnabled && abc {
+            f1()
+        } else {
+            f2()
+        }
+
+        if let v1 = v1, !TestEnum.stale_flag_one.isEnabled && abc {
+            f1()
+        } else if v1 {
+            f2()
+        } else {
+            f3()
+        }
+
+        if v1 {
+            f1()
+        } else if let v1 = v1, !TestEnum.stale_flag_one.isEnabled && abc {
+            f2()
+        } else {
+            f3()
+        }
+
+        if v1 {
```

Comment:
Can we have some comments, and check if they are cleaned up too ? You will have to pass piranha arguments `claenup_comments = True`